profiles/common/modules/features/cce_basic_config/cce_basic_config.install

```diff
@@ -711,6 +712,9 @@ function _cce_basic_config_post_install_default_content_type() {
   foreach ($rdf_mappings as $rdf_mapping) {
     rdf_mapping_save($rdf_mapping);
   }
+
+  // Enable translation of 'body' field by default.
+  multisite_config_service('field')->enableFieldTranslation('body');
```

Comment:
Ho yes, we do want to have this field translatable by default on any new site. And we want to force subsite to have compatibility with this in their development. so imo we need that. We do not want more sites with 'translatable' to FALSE